packages/scheduler/src/Scheduler.js

```diff
@@ -370,3 +447,16 @@ export {
   getCurrentTime as unstable_now,
   forceFrameRate as unstable_forceFrameRate,
 };
+
+export const unstable_startLoggingProfilingEvents = enableProfiling
```

Comment:
This is unfortunate that this adds so many bytes. Don't we have a way to conditionally export?

Reply:
No but we could pull them all in their own namespace.

Reply:
I think we could pull all "advanced features" in a namespace for now.